### DISCUSSION TITLE
Publish macos app, Can not output icns to .app resource folder. Something wrong??

### DISCUSSION THREAD

**Discussion by jim-jiang-github** [AvaloniaUI/Avalonia#11816] [Q&A] [answered]
<img width="1429" alt="image" src="https://github.com/AvaloniaUI/Avalonia/assets/29189410/dddf7bfc-6557-4b2f-b552-81cbeeb04b48">

**Comment by lukaszossowski** [comment, marked as answer]
There is a typo in your path - it si Assets, not Asserts.
